.github/workflows/snyk.yml

```diff
@@ -0,0 +1,19 @@
+name: Snyk
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  security:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+      
+    - name: Run Snyk to check for vulnerabilities
+      uses: snyk/actions/scala@master
+      with:
+        command: monitor
```

Comment:
Mine works without `monitor`. Do we need it? https://github.com/snowplow/enrich/blob/release/1.3.0/.github/workflows/snyk.yml

Reply:
Yes, it defaults to `test` which doesn't upload the report to the Snyk UI. Currently, we're looking at running `monitor` so we have full visibility of our projects in the Snyk UI. 

`test` is useful if you want your build to fail if there are any vulnerabilities, which we will likely add to our build processes in the future once we're more comfortable with where we are at with Snyk.